flink-end-to-end-tests/flink-file-sink-test/src/main/java/org/apache/flink/connector/file/sink/FileSinkProgram.java

```diff
@@ -55,7 +55,6 @@
  * a complete sequence from 0 (inclusive) to 60000 (exclusive).
  */
 public enum FileSinkProgram {
-    ;
```

Comment:
the Jira does not seem to exist. I suggest this is a trivial hot fix, if a committer wants to merge they can, but it is very low priority